referenceFromRuntime.targets

```diff
@@ -36,13 +36,23 @@
       <_filteredReferencePathFromRuntimeByFileName Include="@(_referencePathFromRuntimeByFileName)"
           Condition="'@(_referencePathFromRuntimeByFileName)' == '@(ReferenceFromRuntime)' AND '%(Identity)' != ''" />
 
-      <_missingReferenceFromRuntime Include="@(ReferenceFromRuntime)" Exclude="@(_referencePathFromRuntimeByFileName)" />
+      <_remainingReferenceFromRuntime Include="@(ReferenceFromRuntime)" Exclude="@(_filteredReferencePathFromRuntimeByFileName)" />
+
+      <!-- Fallback and check for native images for the references as well -->
+      <_remainingReferenceFromRuntimeWithNI Include="@(_remainingReferenceFromRuntime->'%(Identity).ni')">
+        <OriginalReferenceFromRuntime>%(Identity)</OriginalReferenceFromRuntime>
```

Comment:
Be careful with this syntax.  I recently used it outside a target and got burned.  Inside a target it does the right thing.  https://github.com/dotnet/corefx/pull/16573#issuecomment-283474053